### PR TITLE
Added JAX-RS dependency and created CarEntity

### PR DESCRIPTION
**Pull request by NFKData**

### CODE REVIEW

**Comment by NFKData** [comment]
[Create Car Entity (should be directly created in DB from JPA), remember persistence.xml](https://trello.com/c/BMO22b6R/8-create-car-entity-should-be-directly-created-in-db-from-jpa-remember-persistencexml)

File: CarAppBackend/src/backend/entity/CarEntity.java [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+	@Column(name="uuid")
+	@Id
+	private String id;
+	
+	@Column(name="brand")
+	private String brand;
+	
+	@Column(name="registration")
+	@Temporal(TemporalType.TIMESTAMP)
+	private Timestamp registration;
```
Comment (javieraviles) [inline CarAppBackend/src/backend/entity/CarEntity.java]: As we are working with JavaSE8 (https://docs.oracle.com/javase/8/docs/api/java/time/LocalDateTime.html), I would prefer `LocalDateTime` over `Timestamp`:

1. Easier diff calculations
2. Much better timezone handling (you will need it)
3. Has nice typing: separation between point-in-time definitions (`Instant`) and duration (`Duration`)
4. Convertions

File: CarAppBackend/src/backend/entity/CarEntity.java [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+	
+	@Column(name="country")
+	private String country;
+	
+	@Column(name="created_at")
+	@Temporal(TemporalType.TIMESTAMP)
+	private Timestamp createdAt;
```
Comment (javieraviles) [inline CarAppBackend/src/backend/entity/CarEntity.java]: Same, go for `LocalDateTime`

File: CarAppBackend/src/backend/entity/CarEntity.java [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+	
+	@Column(name="last_updated")
+	@Temporal(TemporalType.TIMESTAMP)
+	private Timestamp lastUpdated;
```
Comment (javieraviles) [inline CarAppBackend/src/backend/entity/CarEntity.java]: Same, go for `LocalDateTime`

File: CarAppBackend/src/backend/entity/CarEntity.java [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+
+@Entity
+@Table(name="Car")
+public class CarEntity {
```
Comment (javieraviles) [inline CarAppBackend/src/backend/entity/CarEntity.java:15]: Include  `@prepersist` and  `@preupdate` annotations to set createdAt and updatedAt attributes at persisting time.

File: CarAppBackend/src/backend/entity/CarEntity.java (hunk 1 above)
Comment (NFKData) [reply CarAppBackend/src/backend/entity/CarEntity.java]: Changed

File: CarAppBackend/src/backend/entity/CarEntity.java (hunk 2 above)
Comment (NFKData) [reply CarAppBackend/src/backend/entity/CarEntity.java]: Changed

**Review by javieraviles — CHANGES_REQUESTED** [review]
Some changes required on Car entity

File: CarAppBackend/src/backend/entity/CarEntity.java (hunk 3 above)
Comment (NFKData) [reply CarAppBackend/src/backend/entity/CarEntity.java]: Changed

File: CarAppBackend/src/backend/entity/CarEntity.java (hunk 4 above)
Comment (NFKData) [reply CarAppBackend/src/backend/entity/CarEntity.java:15]: Added

**Review by javieraviles — APPROVED** [review]

**Pull request merged**